src/pages/City/Cities/SortedCities.tsx

```diff
@@ -24,7 +24,7 @@ const SortedCities = ( {switcher}: Props) => {
   const [searchedData, setSearchedData] = useState("");
   const [activeUserRoles, setActiveUserRoles] = useState<string[]>([]);
   const {p} = useParams();
-  const [page, setPage] = useState(Number(p));
+  const [page, setPage] = useState(+p);
```

Comment:
Does `p` is the appropriate name for variable?

Reply:
We take this parameter from URL, which matches /clubs/page/:p pattern for simplicity.